sympy/solvers/ode/ode.py

```diff
@@ -2818,7 +2818,7 @@ def _conditional_term_factoring(expr):
     return expr
 
 
-def constant_renumber(expr, variables=None, newconstants=None):
+def constant_renumber(expr, variables=None, newconstants=None, is_system=False):
```

Comment:
Is there any code in sympy that needs to use this with `is_system = False`?

Reply:
@oscarbenjamin Not yet but this may come in handy. Since it gives the end user to choose whether their expressions are independent of each other or not provided if they have passed a list of expressions.

Reply:
If the expressions are independent then they can just do
```
eqs = [constant_renumber(eq) for eq in eqs]
```
I don't see any need to support that case within the function.